src/main/java/xyz/nucleoid/plasmid/util/WoodBlocks.java

```diff
@@ -0,0 +1,31 @@
+package xyz.nucleoid.plasmid.util;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+
+public class WoodBlocks {
+    public static BlockState saplingOf(BlockState state) {
```

Comment:
There should be an additional method for block-to-block conversions. This method should instead copy the state's properties.